clusterloader2/pkg/measurement/common/probes/manifests/dnsLookup/dns-prober-deployment.yaml

```diff
@@ -25,6 +25,8 @@ spec:
             # TODO(oxddr): according to @wojtek-t there are differences between fully and not fully qualified domain names
             # Investigate it and potentially measure latency for both
             - --dns-probe-url=ping-server.probes
+            - --logtostderr=false
+            - --log_file=/var/log/dns-lookup.log
```

Comment:
You need to add a volume and a volumeMount, e.g. see how it's done for hollow node: https://github.com/kubernetes/kubernetes/blob/aacd79cc36347bd56d22169de23a5156761dacd6/test/kubemark/resources/hollow-node_template.yaml#L129